src/screens/ChamberOfSecrets.tsx

```diff
@@ -0,0 +1,237 @@
+/* eslint-disable react-native/no-raw-text */
+/* eslint-disable react/no-unescaped-entities */
+import React, { useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { connect } from 'react-redux';
+
+import {
+  Button,
+  CheckBox,
+  EllipsisText,
+  InputItem,
+  LinearGradient,
+  RadioButton,
+  ScreenTemplate,
+  Text,
+} from 'app/components';
+import { HEADER_HEIGHT } from 'app/components/Header';
+import { ImportWalletType, Wallet } from 'app/consts';
+import { HDSegwitP2SHArWallet, HDSegwitP2SHAirWallet } from 'app/legacy';
+import {
+  createPin as createPinAction,
+  createTxPassword as createTxPasswordAction,
+  createTc as createTcAction,
+} from 'app/state/authentication/actions';
+import { importWallet as importWalletAction } from 'app/state/wallets/actions';
+import { getStatusBarHeight, typography, palette } from 'app/styles';
+
+const i18n = require('../../loc');
+
+const darkSlytherinColor = '#1a472a';
+const lightSlytherinColor = '#2a623d';
+
+const bloodColor = '#8A0303';
+
+const chamberOfSecretsGradient = {
+  colors: [darkSlytherinColor, lightSlytherinColor],
+  start: { x: 1, y: 0 },
+  end: { x: 1, y: 1 },
+};
+
+const buttonLinearGradientProps = { colors: [...chamberOfSecretsGradient.colors].reverse() };
+
+interface Props {
+  createPin: (value: string, { onSuccess }?: { onSuccess?: () => void }) => void;
+  createTxPassword: (value: string, { onSuccess }?: { onSuccess?: () => void }) => void;
+  createTc: (value: boolean) => void;
+  importWallet: (wallet: Wallet, { onSuccess }?: { onSuccess?: () => void }) => void;
+  onButtonPress: () => void;
+  credentials?: {
+    pin: string;
+    password: string;
+  };
+}
+
+const ChamberOfSecrets = (props: Props) => {
+  const {
+    createPin,
+    createTxPassword,
+    createTc,
+    onButtonPress,
+    credentials = { pin: '1234', password: 'qwertyui' },
+  } = props;
+
+  const [isAddWalletChecked, setIsAddWalletChecked] = useState(false);
+  const [walletOptions, setWalletOptions] = useState({
+    type: 'Standard',
+    name: 'My Wallet',
+    seedPhrase:
+      'innocent surge canoe iron nasty dinner aspect bar brand couch input embrace piano example panic champion aerobic debris search fly repeat sand nuclear tuition',
+  });
+
+  const handleButtonPress = () => {
+    // TODO: To implement an import feature here the whole logic must be extracted from ImportWalletScreen first.
+    // addWallet && importWallet({})
+
+    onButtonPress();
+  };
+
+  const onPressSkipTermsConditionsButton = () => {
+    createTc(true);
+
+    handleButtonPress();
+  };
+
+  const onPressSkipOnboardingButton = () => {
+    createTc(true);
```

Comment:
createTc doesn't need boolean to be passed

Reply:
Ups, you're right. Thanks, done.